package.json

```diff
@@ -86,6 +86,8 @@
         "prettier": "^1.18.2",
         "prettier-eslint": "^9.0.0",
         "prettier-eslint-cli": "^5.0.0",
-        "sinon": "^7.3.2"
```

Comment:
This dependency seems to be out-of-date according to the documentation on [npm](https://www.npmjs.com/package/rewire), Please remove it. Also why do you have `slug` as a dependency?